src/io/result_io.jl

```diff
@@ -41,10 +41,10 @@ Matrix{Float64, 2}, of mean and standard deviation for each environmental scenar
 """
 function summarize_env_data(data::AbstractArray)::Array{Float64}
     # TODO: Update once
-    dc_mean = dropdims(mean(data, dims=(1, 2)), dims=(1, 2))'
-    dc_std = dropdims(std(data, dims=(1, 2)), dims=(1, 2))'
-
-    return Array{Float64}(vcat(dc_mean, dc_std))
+    stats_store = Array{Float64}(zeros(2, size(data, 3), size(data, 2)))
+    stats_store[1, :, :] .= Array(dropdims(mean(data; dims=1); dims=1)')
+    stats_store[2, :, :] = Array(dropdims(std(data; dims=1); dims=1)')
```

Comment:
Why on line 45 you are doing `.=` and on line 46 just `=`?

Reply:
Thanks, must have missed pushing it

Reply:
Looking at this, two questions come to mind:

1. Aren't these matrices? (`Matrix()`). If so, I prefer the more explicit `Matrix` over the general `Array`. ?
If it is an `Array`, I'd prefer the number of dimensions be specified.

2. Isn't `data` an array type already? Is the `Array()` necessary?